appengine/brunch/README.md

```diff
@@ -0,0 +1,30 @@
+# Brunch with ES6 running on Google App Engine.
+
+This sample demonstrates how to use [Brunch](http://brunch.io) on
+[Google App Engine Managed VMs](https://cloud.google.com/appengine).
+
+For more information about getting started with Brunch, see the
+[Getting Started with Brunch Guide](https://github.com/brunch/brunch-guide/blob/master/content/en/chapter02-getting-started.md) .
+
+You can also view the [live demo](https://alien-lattice-123714.appspot.com) and read the full [Brunch documentation](https://github.com/brunch/brunch-guide).
```

Comment:
Let me know if you want me to change or remove the link, I don't mind keeping it up.